src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -2,13 +2,14 @@
   (:require-macros [status-im.utils.views :refer [defview]])
   (:require [re-frame.core :refer [subscribe dispatch]]
             [status-im.ui.components.react :refer [view
-                                                text
-                                                image]]
+                                                   text
```

Comment:
Can use move to `:as` usage?

Reply:
I'm kind of careful not to do that to every file I touch, but sure why not.